copyright_headers.go

```diff
@@ -10,4 +12,3 @@ import (
 	_ "github.com/hashicorp/copywrite"
 )
 
-//go:generate go run github.com/hashicorp/copywrite headers
```

Comment:
Is this removal intentional?

Reply:
Yes, I realised line no. 13 wasn't adding any value to the code so I removed it.

Reply:
in retrospect, line no.13 if commented out it will help automate the process. I will just that the line back for convenience!